.github/workflows/release.yml

```diff
@@ -177,6 +177,7 @@ jobs:
         uses: actions/checkout@v3
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
+          ref: v${{ needs.release.outputs.VERSION }}
```

Comment:
Previously this job was checking out the repo at the _time of running this action_, which is _before_ we've built, committed the new version, and pushed those tags. By specifying the tag of this version as the ref, we're making sure the version we just release to npm is being released to Docker.

Reply:
I've tested this change here:
https://github.com/trufflesuite/ganache/actions/runs/3736450089/jobs/6340773768
(I realized I force-pushed to undo the test changes, so I can't point to the code that was used for the test)